<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration_CorrespondenceRejection</class>
 <widget class="QWidget" name="Registration_CorrespondenceRejection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="Registration_Max_Dist_Correspondences">
       <property name="text">
        <string>Max Corres Distance</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_Max_Dist_Correspondences">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>99.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="Registration_OnetoOne">
     <property name="text">
      <string>OnetoOne</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="Registration_MedianDistance">
       <property name="text">
        <string>Median Distance</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="Registration_MedianFactor"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="Registration_SampleConsensus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Sample Consensus</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Max iteration</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="Registration_SAC_MaxIteration">
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>InlierThresh</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="Registration_SAC_InlierThreshold">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="maximum">
          <double>100000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.001000000000000</double>
         </property>
         <property name="value">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="Registration_Trimmed">
       <property name="text">
        <string>Trimmed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_Trimmed">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="Registration_VarTrimmed">
     <property name="text">
      <string>VarTrimmed</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_GetList">
     <property name="text">
      <string>GetList</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>Registration_SampleConsensus</tabstop>
  <tabstop>Registration_SAC_MaxIteration</tabstop>
  <tabstop>Registration_SAC_InlierThreshold</tabstop>
  <tabstop>Registration_VarTrimmed</tabstop>
  <tabstop>Registration_Max_Dist_Correspondences</tabstop>
  <tabstop>doubleSpinBox_Max_Dist_Correspondences</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
